influxdb/client.py

```diff
@@ -62,11 +62,26 @@ class InfluxDBClient(object):
     :type udp_port: int
     :param proxies: HTTP(S) proxy to use for Requests, defaults to {}
     :type proxies: dict
+    :param uri: uri to use in URL when using a reverse proxy, defaults to ''
+        example https://myserver.domain.com/influxproxy with an Apache 
+        reverse proxy config that should look like :
+            <Proxy http://localhost:8086/*>
+                 Allow from all
+            </Proxy>
+            <LocationMatch "/influxproxy">
+                 ProxyPass http://localhost:8086 retry=0
+                 ProxyPassReverse http://localhost:8086
+                 RequestHeader set X-Forwarded-For %{REMOTE_ADDR}s
+            </LocationMatch>
+
+    :type uri: str
+
     """
 
     def __init__(self,
                  host='localhost',
                  port=8086,
+                 uri='',
```

Comment:
Note that this parameter is not used when `use_udp` is set. But I guess this is fine since that's also true for some other parameters.